tests/network/localnet/test_ovs_bridge.py

```
@@ -12,3 +53,26 @@ def test_connectivity_over_migration_between_ovs_bridge_localnet_vms(
+@pytest.mark.usefixtures("nncp_localnet_on_secondary_node_nic")
+@pytest.mark.polarion("CNV-11905")
+def test_connectivity_after_interface_state_change_in_ovs_bridge_localnet_vms(
+    vm_ovs_bridge_localnet_no_ip, vm_ovs_bridge_localnet_2
+):
+    run_vms(vms=(vm_ovs_bridge_localnet_no_ip, vm_ovs_bridge_localnet_2))
+    link_state = get_link_state_of_interface(vm_ovs_bridge_localnet_no_ip, LOCALNET_OVS_BRIDGE_NETWORK)
+    assert link_state == "down"
+    patch_localnet_interface(vm_ovs_bridge_localnet_no_ip, "up")
+    link_state = get_link_state_of_interface(vm_ovs_bridge_localnet_no_ip, LOCALNET_OVS_BRIDGE_NETWORK)
+    assert link_state == "up"
+    result = vm_ovs_bridge_localnet_no_ip.console(commands=["ip link show dev eth0"], timeout=_DEFAULT_CMD_TIMEOUT_SEC)
```

Flake8 (7.2.0)

[error] local variable 'result' is assigned to but never used

(F841)